src/js/dashboard/dashboard.controller.js

```diff
@@ -6,27 +6,32 @@ function DashboardComponent(franklinAPIService, $scope,
   /* jshint validthis: true */
   const dc = this;
 
-  //variables
-  dc.username = '';
-  dc.showLoader = false;
-  dc.deployedRepos = [];
+  let variables = {
+    username: '',
+    showLoader: false,
+    deployedRepos: [],
+  }
+
+  let functions = {
```

Comment:
Pro tip: With ES6, if object keys and values are the the same name you can just list them directly

```diff
let functions = {
-    getFranklinRepos: getFranklinRepos,
-    getDeployableRepos: getDeployableRepos,
-    listFranklinRepos: listFranklinRepos,
-    listDeployableRepos: listDeployableRepos,
-    logout: logout,
-    showRepoDetail: showRepoDetail,
-    error: error
}

let functions = {
+    getFranklinRepos,
+    getDeployableRepos,
+    listFranklinRepos,
+    listDeployableRepos,
+    logout,
+    showRepoDetail,
+    error
}
```